addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/UniFiController.java

```diff
@@ -105,6 +106,20 @@ public void logout() throws UniFiException {
         executeRequest(req);
     }
 
+    public void blockStation(UniFiClient client, boolean block, String siteName) throws UniFiException {
+        UniFiControllerRequest<UniFiClient[]> req = newRequest(UniFiClient[].class);
```

Comment:
I think the following was the idea: Since the return value is not used, it's not needed to parse the returned json either: 
```suggestion
        UniFiControllerRequest<Void> req = newRequest(Void.class);
```

Reply:
But of course! Updated.